openxla/openxla-pjrt-plugin
# implement a channel provider with a file-based key value store

Provides a channel provider with a file-based key value store.

How to test:
1. The test needs two GPUs.
2. NCCL 2.18.1-1. Install it from https://github.com/NVIDIA/nccl.
3. Open two terminals, go to `test` and in each terminal, run `run0.sh` and `run1.sh`, respectively.

`run0.sh` should show the result `[2.]` since 1 + 1 = 2.
